docs/identityManager.md

```diff
@@ -17,6 +17,28 @@ The IdentityManager should be able to perform the following actions:
 ## Design
 The IdentityManager contract has any number of owners and one recovery key for each proxy. It also has rate limits for each caller on some of the functions. Proxies can be created, transfered to and from the IdentityManager. Owners can be added and removed, the recovery key can be changed.
 
+### Values of constants
+There are three constants that needs to be set when creating this contract. All of these constants are specified in seconds.
+```
+_userTimeLock - Time before new owner added by recovery can control proxy
+_adminTimeLock - Time before new owner can add/remove owners
+_adminRate - Time period used for rate limiting a given key for admin functionality
```

Comment:
We need to add some more info about what `adminRate` is.

Reply:
Will add more info